tests/test_icecream.py

```diff
@@ -467,6 +492,18 @@ def testIncludeContextMultiLine(self):
         pair = parseOutputIntoPairs(out, err, 3)[1][0]
         assert pair == ('multilineStr', ic.argToStringFunction(multilineStr))
 
+    def testContextAbsPathMultiLine(self):
+        multilineStr = 'line1\nline2'
+        with configureIcecreamOutput(includeContext=True, contextAbsPath=True):
+            with disableColoring(), captureStandardStreams() as (out, err):
+                ic(multilineStr)
+
+        firstLine = err.getvalue().splitlines()[0]
+        assert lineIsAbsPathContext(firstLine) # TODO
```

Comment:
quick question: what remains TODO here? or is this an old comment and no longer applicable? 🙂